ff2mpv.js

```diff
@@ -2,6 +2,17 @@ function ff2mpv(url) {
     browser.runtime.sendNativeMessage("ff2mpv", { url: url });
 }
 
+function notifyContentScript(e) {
```

Comment:
This function takes a parameter, but you call it without one below. Is that intentional?